packages/tools/devtools/devtools-example/jest.config.js

```diff
@@ -2,16 +2,28 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+// Get the test port from the global map and set it in env for this test
+const testTools = require("@fluidframework/test-tools");
+const { name } = require("./package.json");
+
+const mappedPort = testTools.getTestPort(name);
+process.env["PORT"] = mappedPort;
 
 module.exports = {
-	preset: "ts-jest",
+	preset: "jest-puppeteer",
 	roots: ["<rootDir>/src"],
+	globals: {
+		PATH: `http://localhost:${mappedPort}`,
+	},
 	transform: {
-		"^.+\\.tsx?$": "ts-jest",
+		"^.+\\.(ts|tsx)?$": "ts-jest",
```

Comment:
This change isn't correct. It will match files with no extension, in addition to ts and tsx files. What was there before was correct and would match on both ts and tsx files.

Reply:
```suggestion
		"^.+\\.tsx?$": "ts-jest",
```